src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -51,7 +51,30 @@ internal partial class Binder
                     // BindType for AttributeSyntax's name is handled specially during lookup, see Binder.LookupAttributeType.
                     // When looking up a name in attribute type context, we generate a diagnostic + error type if it is not an attribute type, i.e. named type deriving from System.Attribute.
                     // Hence we can assume here that BindType returns a NamedTypeSymbol.
-                    boundAttributeTypes[i] = (NamedTypeSymbol)binder.BindType(attributeToBind.Name, diagnostics).Type;
+                    var boundType = binder.BindType(attributeToBind.Name, diagnostics);
+                    var boundTypeSymbol = (NamedTypeSymbol)boundType.Type;
+
+                    // Check type arguments inside the attribute type (unless the attribute type is already an error).
+                    if (boundTypeSymbol.TypeKind != TypeKind.Error)
+                    {
+                        var location = attributeToBind.GetLocation();
```

Comment:
>attributeToBind [](http://example.com/codeflow?start=39&length=15)

Location probably should be just the type rather than the entire attribute syntax

Reply:
Right, that looks better. Fixed: 7a4b202cad2947b3be6e0b2ebdad561dc80e455d